content/docs/command-reference/check-ignore.md

```diff
@@ -0,0 +1,83 @@
+# check-ignore
+
+Used for debugging `.dvcignore` / exclude files.
+
+## Synopsis
+
+```usage
+usage: usage: dvc check-ignore [-h] [-q | -v] [-d] [-n]
+                               targets [targets ...]
+
+positional arguments:
+  targets        Input files/directories to check ignore patterns.
+```
+
+## Description
+
+For each pathname given via the command-line , check whether the file is excluded 
+by `.dvcignore` output the path if it is excluded.
```

Comment:
``This helper command checks whether the given `targets` are ignored by DVC according to the [`.dvcignore` file](/doc/user-guide/dvcignore) (if any). The ones that are indeed ignores are printed back.``

Reply:
Let's include a link to the user guide Dvcignore section so that users can see the logic behind it. It would be great to mention this new command in that section as well.

Reply:
OK, it's in my suggestion:
```suggestion
This helper command checks whether the given `targets` are ignored by DVC
according to the [`.dvcignore` file](/doc/user-guide/dvcignore) (if any). The
ones that are indeed ignores are printed back.
```
Committing.

Reply:
- [ ] It would be great to mention this new command in that section as well.

Indeed. Unresolving...